src/v/config/base_property.cc

```diff
@@ -30,4 +30,15 @@ std::ostream& operator<<(std::ostream& o, const base_property& p) {
     return o;
 }
 
+std::string_view visibility_string(visibility v) {
```

Comment:
nit: I've been calling this `to_string_view`, and it can be `constexpr`.

I think gcc will complain about no return value.

Reply:
I never liked this name anyway... changed.